src/scenes/NftDetailPage/NftDetailVideo.tsx

```diff
@@ -31,13 +31,13 @@ function NftDetailVideo({ mediaRef, maxHeight }: Props) {
       loop
       playsInline
       controls
-      onLoadStart={setContentIsLoaded}
+      onLoadedData={setContentIsLoaded}
```

Comment:
changed `onLoadStart` => `onLoadedData` as the latter waits until the video has completed loading